lib/infrastructure/pigeons/pigeons.g.dart

```diff
@@ -1039,12 +1039,37 @@ class KeepUnusedHack {
   }
 }
 
+abstract class RawIncomingPacketsCallbacks {
+  void onPacketReceived(ListWrapper arg);
+
+  static void setup(RawIncomingPacketsCallbacks api) {
+    {
+      const BasicMessageChannel<Object> channel = BasicMessageChannel<Object>(
+          'dev.flutter.pigeon.RawIncomingPacketsCallbacks.onPacketReceived',
+          StandardMessageCodec());
+      if (api == null) {
+        channel.setMessageHandler(null);
+      } else {
+        channel.setMessageHandler((Object message) async {
+          assert(message != null,
```

Comment:
I like that pattern 👍

Reply:
Although on second thought, `assert` will throw in debug environment only, what should happen in release? Right now it will throw Error, do we want that? Could we handle it nicer? Just thinking out aloud...